packages/cysync-core/src/pages/MainApp/Components/AssetAllocation.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  TableNameBox,
+  TableBody,
+  Table,
+  TableDataRow,
+  TableHeader,
+  TableHeaderData,
+  TableTitle,
+  Typography,
+  AllocationShare,
+} from '@cypherock/cysync-ui';
+import React, { useEffect, useState } from 'react';
+
+import {
+  AssetIconNameBox,
+  AssetVariants,
+} from '~/pages/MainApp/Components/AssetIconNameBox';
+
+type TableHeaderNames = 'Asset' | 'Price' | 'Amount' | 'Value' | 'Allocation';
+export type NameVariants = 'Bitcoin' | 'Ethereum';
```

Comment:
Duplicate of the type defined in `AssetIconNameBox`. Also, it'll be better if we used the `id` from `packages/coins` package.